BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -18,6 +18,7 @@ IMPROVEMENTS / FIXES
 - UI:
 	- Team Icon text now scales with icon scale.
 - AI:
+	Fix AI not going into Ramming mode when guns possess ammo, but not enough to meet resourceRequestAmount.
```

Comment:
Is this just for the orbital AI or for all AIs?

Were all of these actually in v1.7.1.0 or are they new (in which case, they should be in the new improvements/fixes)?
Typos:
- differeing -> differing
- absed -> based
- weaposn -> weapons
- Decharge/uncharge -> Discharge (unless it's supposed to be French)

Reply:
For anything that is using HasWeaponsAndAmmo(), so all AIs that use it to determine if they should go into ramming mode would be affected.
The ``cruiseRangeTrigger`` is technically old and was added a version or two ago, but never made it into the changleog until now.